packages/teams-js/src/public/tasks.ts

```diff
@@ -72,7 +72,9 @@ export namespace tasks {
    * Submit the task module.
    *
    * @param result - Contains the result to be sent to the bot or the app. Typically a JSON object or a serialized version of it
-   * @param appIds - Helps to validate that the call originates from the same appId as the one that invoked the task module
+   * @param appIds - Valid application/s that the dailog can send result back to once dialog is submitted. It pervents malicious apps to get the dialog result.
```

Comment:
Make sure you update this to be the same as the comment in dialog once you change the wording to match what erika suggested